dirsrvtests/tests/suites/clu/dsidm_posixgroup_test.py

```diff
@@ -0,0 +1,425 @@
+    modify(standalone, DEFAULT_SUFFIX, topology_st.logcap.log, args, warn=False)
+    check_value_in_log_and_reset(topology_st, check_value=output)
+    assert test_posixgroup.present('description', 'new_posixgroup_description')
+
+    log.info('Test dsidm posixgroup modify delete')
+    args.changes = ['delete:description:new_posixgroup_description']
+    modify(standalone, DEFAULT_SUFFIX, topology_st.logcap.log, args, warn=False)
+    check_value_in_log_and_reset(topology_st, check_value=output)
+    assert not test_posixgroup.present('description', 'new_posixgroup_description')
+
+
+@pytest.mark.skipif(ds_is_older("1.4.2"), reason="Not implemented")
+def test_dsidm_posixgroup_list(topology_st, create_test_posixgroup):
+    """ Test dsidm posixgroup list option
+
+    :id: 03b72540-2c2b-405d-a918-4ca0dbddfcc5
+    :setup: Standalone instance
+    :steps:
+        1. Run dsidm posixgroup list option without json
+        2. Check the output content is correct
+        3. Run dsidm posixgroup list option with json
+        4. Check the output content is correct
+        5. Delete the posixgroup
+        6. Check the posixgroup is not in the list with json
+        7. Check the posixgroup is not in the list without json
+    :expectedresults:
+        1. Success
+        2. Success
+        3. Success
+        4. Success
+        5. Success
+        6. Success
+        7. Success
+    """
+
+    standalone = topology_st.standalone
+    args = FakeArgs()
+    args.json = False
+    json_list = ['type',
+                 'list',
+                 'items']
+
+    log.info('Empty the log file to prevent false data to check about posixgroup')
+    topology_st.logcap.flush()
+
+    log.info('Test dsidm posixgroup list without json')
+    list(standalone, DEFAULT_SUFFIX, topology_st.logcap.log, args)
+    check_value_in_log_and_reset(topology_st, check_value=posixgroup_name)
+
+    log.info('Test dsidm posixgroup list with json')
+    args.json = True
+    list(standalone, DEFAULT_SUFFIX, topology_st.logcap.log, args)
+    check_value_in_log_and_reset(topology_st, content_list=json_list, check_value=posixgroup_name)
+
+    log.info('Delete the posixgroup')
+    posixgroups = PosixGroups(standalone, DEFAULT_SUFFIX)
+    test_posixgroup = posixgroups.get(posixgroup_name)
+    test_posixgroup.delete()
+
+    log.info('Test empty dsidm posixgroup list with json')
+    list(standalone, DEFAULT_SUFFIX, topology_st.logcap.log, args)
+    check_value_in_log_and_reset(topology_st, content_list=json_list, check_value_not=posixgroup_name)
+
+    log.info('Test empty dsidm posixgroup list without json')
+    args.json = False
+    list(standalone, DEFAULT_SUFFIX, topology_st.logcap.log, args)
+    check_value_in_log_and_reset(topology_st, check_value_not=posixgroup_name)
+
+
+@pytest.mark.xfail(reason="Will fail because of DS6439")
+@pytest.mark.skipif(ds_is_older("1.4.2"), reason="Not implemented")
+def test_dsidm_group_get_dn(topology_st, create_test_group):
```

Comment:
Probably, it makes sense to rename the test function to `test_dsidm_posixgroup_get_dn` for consistency